src/main/index.js

```diff
@@ -928,6 +928,14 @@ function runApp() {
     return app.getPath('pictures')
   })
 
+  ipcMain.on(IpcChannels.REQUEST_FULLSCREEN, ({ sender }) => {
+    sender.executeJavaScript('document.getElementById("videoContainer").requestFullscreen({navigationUI: "hide"})', true)
```

Comment:
Question: why is this required? (`ipcRenderer.send(IpcChannels.REQUEST_FULLSCREEN)` then this

Reply:
[Transient activation](https://developer.mozilla.org/en-US/docs/Web/Security/User_activation#transient_activation) is required to trigger fullscreen as per the Fullscreen API, i.e., programmatic interaction will not work. Electron's [`executeJavascript` function](https://www.electronjs.org/docs/latest/api/web-contents#contentsexecutejavascriptcode-usergesture), specifically with the `userGesture` boolean set to true, is a clean way for us to bypass this restriction.

Reply:
Do you mind adding code comments for those 2 (I guess FT won't have too many of these) to indicate they are workaround (not usual practice)

Reply:
Added comments now